src/lib.rs

```diff
@@ -309,12 +309,30 @@ mod tests {
             break;
         }
     }
+    
     #[test]
     #[serial]
     fn test_get_version() {
         HWIClient::get_version().unwrap();
     }
 
+    #[test]
+    #[serial]
+    #[ignore]
+    fn test_setup_device() {
+        let devices = HWIClient::enumerate().unwrap();
+        let unsupported = ["ledger", "coldcard", "jade"];
+        for device in devices {
+            let device = device.unwrap();
```

Comment:
Ok, we need find_device function instead of enumerate for not initialized devices. I'll open a new PR to add find_device function. 
I did some tests and I saw that the setup_device command does not work on emulators but only on physical devices (at least my emulator returns the error "PIN requested but no sequence was configured").

Reply:
> Ok, we need find_device function instead of enumerate for not initialized devices. I'll open a new PR to add find_device function.

This would be great, thanks!

> I did some tests and I saw that the setup_device command does not work on emulators but only on physical devices (at least my emulator returns the error "PIN requested but no sequence was configured").

I'm hitting the same error, I just opened https://github.com/bitcoin-core/HWI/issues/646 :)